README.md

```diff
@@ -4,6 +4,9 @@
   <p align="center">Build, develop, and deploy Next.js apps for Cloudflare Pages.</p>
 </p>
 
+> [!NOTE]
+> You may want to consider using [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare), which allows you to build and deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/static-assets/), use [Node.js APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/) that Cloudflare Workers supports, and enables additional Next.js features like Incremental Static Regeneration.
```

Comment:
I feel like this should say that it's the recommended method considering Pages will be slowly phased out 🤷

Reply:
I am not sure we are quite at the place to say that `@opennextjs/cloudflare` is the recommended solution; it is still pre-alpha. But I agree that once it begins to stabilize, this is the message we should be sending.